test/config/pool-mock-tp.toml

```diff
@@ -1,5 +1,5 @@
 listen_address = "0.0.0.0:34254"
-tp_address = "75.119.150.111:8442"
+tp_address = "127.0.0.1:8442"
```

Comment:
It might be good to add a comment here that this should always point at localhost.

Reply:
I added a line, but I sucks at writing comments, so please check again if is fine now!